pan3d/viewer.py

```diff
@@ -0,0 +1,265 @@
+import json
+import os
+import pyvista
+from pvxarray.vtk_source import PyVistaXarraySource
+import xarray
+
+from trame.decorators import TrameApp, change
+from trame.ui.vuetify import SinglePageWithDrawerLayout  # TODO: upgrade to vuetify 3
```

Comment:
The PyVista gui from trame are based on vue2 (They use vuetify2 for their buttons) so you will run into issue.

Reply:
Yes, I tested this and did get a working version with a quick fix in the pyvista code. I filed an issue for the pyvista developers here: https://github.com/pyvista/pyvista/issues/4776. I have a vuetify 3 version of the ui that would work if we can help the pyvista folks address this (hopefully in a more robust manner than my quick fix). But for now I'll leave a todo.